### DISCUSSION TITLE
Multiple dashboards?

### DISCUSSION THREAD

**Discussion by adabelleleiram** [UI-Lovelace-Minimalist/UI#576] [Q&A] [answered]
Is it possible to create multiple dashboards with this integration? I've tried multiple things, e.g. new yaml files in dashboard folder, adding another instance of the ULM integration, adding a new dashboard manually in the configuration file but nothing has worked so far. This would be super useful for separate e.g. mobile, tablet, guest dashboards

**Comment by AndyVRD** [comment, marked as answer]
To add more dashboards configure them in your "configuration.yaml" file, see example below. You can always check the git repo from [Bas](https://github.com/basbruss/Minimalist-Dashboards), he added some examples that can be a inspiration for building your dashboard(s). 

    lovelace:
      mode: storage
      dashboards:
        lovelace-minimalist:
          mode: yaml
          title: Minimalist
          icon: mdi:flower
          show_in_sidebar: true
          filename: ui_lovelace_minimalist/dashboard/ui-lovelace.yaml
        lovelace-minimalist-xxxxx:
          mode: yaml
          title: Minimalist xxxxx
          icon: mdi:flower
          show_in_sidebar: true
          filename: ui_lovelace_minimalist/dashboard/ui-lovelace-xxxxx.yaml

**Reply by adabelleleiram** [reply]
I tried that first but couldn't get it to work. Took a fresh pair of eyes to notice the typo in one of my paths 😅 Thanks for the help!
